src/components/HomeSearchBox/index.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { useState } from 'react'
+import useTranslation from 'next-translate/useTranslation'
+
+const HomeSearchBox = () => {
+  const { t } = useTranslation('home')
+  const [searchItem, setSearch] = useState('')
+
+  const handleClickSearch = (e) => {
+    window.open(`products/${searchItem}`)
+    setSearch("")
```

Comment:
Don't forget to use single quote.
```suggestion
    setSearch('')
```